pkg/server/diagnostics/reporter.go

```diff
@@ -46,6 +49,11 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+// TelemetryHttpTimeout allows for configuration of the client timeout
+// for sending telemetry reports. It is not expected that customers
+// whould tweak this.
+var TelemetryHttpTimeout = envutil.EnvOrDefaultDuration("COCKROACH_TELEMETRY_HTTP_CLIENT_TIMEOUT", 5*time.Minute)
```

Comment:
nit: can this be private?

Reply:
I modify this in the test which is in a separate package

Reply:
ack